### DISCUSSION TITLE
Ability to add custom shipnames

### DISCUSSION THREAD

**Discussion by flygfantast** [jvde-github/AIS-catcher#89] [Ideas]
I would like to have the ability to add custom shipnames.
For example on the attached picture the 2 last entries are base stations, and never sends out a ship name. It would be nice if I could "name" them myself. Also the SAR aircraft would be nice to label.

Maybe some similar mapping that we do on the ais2adsb project? But here you map MMSI to a "name".

{ 2655085:"Nkpg" , 2655087:"Ervasteby"  }

![image](https://user-images.githubusercontent.com/124347186/224470779-f8e0e8f9-8513-4b7a-98c4-cf9cb53554d4.png)

**Comment by jvde-github** [comment]
The easiest approach is to use a plugin like in the link below and edit the table. You can overwrite ship name and/or the callsign. 
To create your own overview you can edit the shipNames object which has already to entries to give an example. 

https://github.com/jvde-github/AIS-Catcher-PLUGINS/blob/b20e5d8265e95939b435c80b2d3449ecf43391ff/shipnames.pjs

Hope that helps

**Reply by flygfantast** [reply]
Thanks! It worked nicely :)

**Reply by jvde-github** [reply]
Good to hear! Thanks
